src/router.php

```diff
@@ -2,25 +2,51 @@
 
 namespace Bugcache;
 
-use Aerys as aerys;
+use Aerys;
```

Comment:
Can you please leave this lowercase? That'll make it easy to differ between imported namespaces and subnamespaces. (also no shift to type :-P)